src/alpine/3.17/helix/amd64/Dockerfile

```diff
@@ -1,33 +1,52 @@
-FROM mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.17-local
+FROM alpine:3.17
+# Install .NET and test dependencies
 RUN apk update && apk add --no-cache \
-    cargo \
+    bash \
+    coreutils \
+    curl \
+    icu-data-full \
     iputils \
-    libffi-dev \
+    lttng-ust \
     musl-locales \
-    rust
+    openssl \
+    python3 \
+    sudo \
+    tzdata
 
 # Install Helix Dependencies
-RUN ln -sf /usr/bin/python3 /usr/bin/python && \
```

Comment:
Helix agents expect to just find "python" as the entry point and value for `$HELIX_PYTHONPATH` given the bottom of this file it seems like that doesn't work.  Rest of the helix-y changes seem fine.

Reply:
I was no intention, I'll put it back. I updated the rest so we do not rely on the link in case we fix/change the HELIX_PYTHONPATH one day.

Reply:
hmm. actually, it is just moved to line 20. so everything should work as it did before. I put the `apk add` first to preserve the sequence but I guess in this case it does not matter.